example/test_driver/bench_test.dart

```diff
@@ -14,7 +14,8 @@ Future<void> main() async {
     final Timeline timeline = await driver.traceAction(() async {
       await driver.scroll(
         view,
-        0, -3400,
+        0,
```

Comment:
I guess `flutter format .` caught this, since I didn't touch it.

Reply:
It changes sometimes depending on the flutter version, no worries.